src/themes/ec/_variables.scss

```diff
@@ -547,23 +547,23 @@ $file-upload: (
 
 // Gallery
 $gallery: (
-  actions-margin: map.get($spacing, 'm'),
-  actions-margin-desktop: map.get($spacing, 'm'),
-  arrow-margin: map.get($spacing, 'm'),
-  arrow-margins: map.get($spacing, 'xs') map.get($spacing, '2xs'),
-  close-button-color: map.get($color, 'white-100'),
-  description-background: rgba(map.get($color, 'black'), 0.65),
-  description-icon: map.get($icon, 'xs'),
-  description-icon-mobile: map.get($icon, 'xs'),
+  actions-margin: var(--s-xl),
+  actions-margin-desktop: var(--s-xs),
+  arrow-margin: var(--s-m),
+  arrow-margins: var(--s-xs) var(--s-2xs),
+  close-button-color: map.get($color, 'white'),
+  counter-border: 1px solid map.get($color, 'white-100'),
+  counter-font-weight: map.get($font-weight, 'regular'),
+  description-background: rgba(map.get($color, 'dark-100'), 0.7),
```

Comment:
we should use the newly introduced css variables for overlay colors (--c-ov-l and --c-ov-d)

Reply:
mmh..these colors are only available in EC, anyway, i've used the dark color as the background, not sure how and where to use the light color instead. I've simplified the variables, the styles are pretty similar only a few of them were still used to set different styles in the two themes.